src/api/spec/features/beta/webui/requests_index_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'browser_helper'
+
+RSpec.describe 'Requests Index' do
+  let(:submitter) { create(:confirmed_user, :with_home, login: 'kugelblitz') }
+  let(:receiver) { create(:confirmed_user, :with_home, login: 'titan') }
+  let(:target_project) { receiver.home_project }
+  let(:target_package) { create(:package, name: 'goal', project_id: target_project.id) }
+  let(:source_project) { submitter.home_project }
+  let(:source_package) { create(:package, name: 'ball', project_id: source_project.id) }
+
+  let!(:bs_requests) do
+    create_list(:bs_request_with_submit_action, 6,
+                description: 'Please take this',
+                creator: submitter,
+                target_package: target_package,
+                source_package: source_package)
+
+    create_list(:bs_request, 4,
```

Comment:
This makes little sense I think, you are not testing different types.

Reply:
We will soon, but this is out of this PR scope so, right, I'm removing it.